timev.rb

```diff
@@ -1,20 +1,62 @@
-# Time is an abstraction of dates and times. Time is stored internally as
-# the number of seconds with subsecond since the _Epoch_,
-# 1970-01-01 00:00:00 UTC.
-#
-# The Time class treats GMT
-# (Greenwich Mean Time) and UTC (Coordinated Universal Time) as equivalent.
+# A \Time object represents a date and time:
+#
+#   Time.new(2000, 1, 1, 0, 0, 0) # => 2000-01-01 00:00:00 -0600
+#
+# Although its value is stored internally as a single numeric
+# (see {Epoch Seconds}[rdoc-ref:Time@Epoch+Seconds] below),
+# it can be convenient to deal with the value by parts:
+#
+#   t = Time.new(-2000, 1, 1, 0, 0, 0.0)
+#   # => -2000-01-01 00:00:00 -0600
+#   t.year # => -2000
+#   t.month # => 1
+#   t.mday # => 1
+#   t.hour # => 0
+#   t.min # => 0
+#   t.sec # => 0
+#   t.subsec # => 0
+#
+#   t = Time.new(2000, 12, 31, 23, 59, 59.5)
+#   # => 2000-12-31 23:59:59.5 -0600
+#   t.year # => 2000
+#   t.month # => 12
+#   t.mday # => 31
+#   t.hour # => 23
+#   t.min # => 59
+#   t.sec # => 59
+#   t.subsec # => (1/2)
+#
+# == Epoch Seconds
+#
+# The value of a \Time object is stored internally as a Rational object
```

Comment:
I would guess that is not always true, for  `time_to_r` in `time.c` has this code:

```c
    GetTimeval(time, tobj);
    v = rb_time_unmagnify_to_rational(tobj->timew);
    if (!RB_TYPE_P(v, T_RATIONAL)) {
        v = rb_Rational1(v);
    }    
    return v;
```

So I'm guessing there is at least a possibility that that is not stored as a rational always.  My guess would be that it is stored as an integer if there are no subseconds.

In any case, I don't think it makes sense to discuss implementation details such as how values are stored internally.  You can mention that `to_r` should always provide an exact time, whereas `to_i` and `to_f` may not for some times.  Other documentation changes that mentions how values are stored internally should also be updated.

Reply:
Done (two places).